### ISSUE TITLE
Mountebank does not update the decorated response value as expected.

### ISSUE THREAD

**Issue by nottyo** [bbyars/mountebank#173]
#### Expected behaviour
Mountebank should update the value in response message with the condition defined in behaviors.

#### Actual behaviour
Mountebank always return the first decorated response.

#### Steps to reproduce
1. Create new imposters with response decoration
   This mock substitutes an "**${id}**" in response from request query e.g. **_\<requestId\>100\</requestId\> \<requestId\>200\</requestId\>_**

    POST: http://localhost:2525/imposters

        {
        	"port": "10500",
        	"protocol": "http",
        	"stubs": [
        		{
        			"responses": [
        				{
        					"is": {
        						"statusCode": 200,
        						"headers": {
        							"content-type": "text/xml; charset=utf-8"
        						},
        						"body": "<requestId>${id}</requestId>"
        					},
        					"_behaviors": {
        						"decorate": "function (request, response) { response.body = response.body.replace('${id}', request.query['id']);}"
        					}
        				}
        			],
        			"predicates": [
        				{
        					"and": [
        						{
        							"exists": {
        								"query": {
        									"id": true
        								}
        							}	
        						},
        						{
        							"equals": {
        								"path": "/test",
        								"method": "GET"
        							}
        						}
        					]
        					
        				}
        			]
        		}
        	]
        }

2. Execute

```
curl http://localhost:10500/test?id=100
curl http://localhost:10500/test?id=200
curl http://localhost:10500/test?id=300
curl http://localhost:10500/test?id=400
curl http://localhost:10500/test?id=500
```

3. Then you get

```
<requestId>100</requestId>
<requestId>100</requestId>
<requestId>100</requestId>
<requestId>100</requestId>
<requestId>100</requestId>
```

4. Expected behavior is

```
<requestId>100</requestId>
<requestId>200</requestId>
<requestId>300</requestId>
<requestId>400</requestId>
<requestId>500</requestId>
```

This issue has been happening since Mountebank 1.7.0 
...

#### Software versions used

```
OS         : OSX 10.11.5 (El Capitan)
mountebank : 1.7.0
node.js    : 7.0.0
  (only if installed via npm)
Installation method : npm
  (npm, zip, tar, pkg, deb, rpm)
```

#### Log contents in mb.log when running mb --loglevel debug

```
info: [mb:2525] mountebank v1.7.0 now taking orders - point your browser to http://localhost:2525 for help
info: [mb:2525] POST /imposters
info: [http:10500] Open for business...
info: [http:10500] ::1:53274 => GET /test?id=100
info: [http:10500] ::1:53311 => GET /test?id=200
info: [http:10500] ::1:53348 => GET /test?id=300
```

**Comment by bbyars** [comment]
Thanks for reporting. It appears the workaround, until I can sort out what's happening, is to return a new response out of your decorate function:

```
function (request, response) {
    return {
        body: response.body.replace('${id}', request.query['id']),
        statusCode: response.statusCode,
        headers: response.headers
    };
```

**Comment by bbyars** [comment]
Fixed as of 5acd741, will release patch today

**Comment by bbyars** [comment]
v1.7.2 released with the fix. Sorry for the hassle
